inc/fragments/prefix.php

```diff
@@ -47,4 +106,24 @@ public function elements() {
 		return array_merge( self::PREFIXES, $this->translatable_prefixes() );
 	}
 
+	/**
+	 * Return an array of prefixes that don't have spaces.
+	 *
+	 * @since  0.2
+	 * @todo   If there's a better way to do this that's faster, it would be great to implement that.
+	 * @return array Prefixes that don't have spaces.
+	 */
+	public static function no_space_prefixes() {
```

Comment:
This could be slimmed down a bit, but not necessarily faster. ```$suffixes = self::SUFFIXES;
return array_filter( $suffixes, function ( $suffix ) {
    return ! strpos( $suffix, ' ' ) === false;
} );```